lib/dpl/provider/bluemix_cloudfoundry.rb

```diff
@@ -0,0 +1,22 @@
+module DPL
+  class Provider
+    class BluemixCF < CloudFoundry
```

Comment:
I am open to better/more consistent names with other dpl providers. I chose this name to leave open the opportunity for deploy providers to work with other Bluemix such as the container service. Would bluemix-cloudfoundry be more consistent?